C/CasADi/build_tarballs.jl

```diff
@@ -0,0 +1,81 @@
+using BinaryBuilder, Pkg
+
+name = "CasADi"
+
+version = v"3.5.5"
+
+sources = [
+    GitSource(
+        "https://github.com/casadi/casadi.git",
+        "fadc86444f3c7ab824dc3f2d91d4c0cfe7f9dad5",
+    ),
+    DirectorySource("./bundled"),
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/casadi
+install_license LICENSE.txt
+
+mkdir -p build
+cd build
+
+export CXXFLAGS="${CXXFLAGS} -std=c++11"
+export CFLAGS="${CFLAGS} -fPIC"
+
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DWITH_IPOPT=ON \
+    -DWITH_EXAMPLES=OFF \
+    ..
+
+if [[ "${target}" == *-linux-* ]]; then
+        make -j ${nproc}
+else
+    if [[ "${target}" == *-mingw* ]]; then
+        cmake --build . --config Release
+    else
+        cmake --build . --config Release --parallel
+    fi
+fi
```

Comment:
I'm missing the difference between the first and the last branch.   I presume the Windows build had problems compiling in parallel?

Reply:
I just copied it from some of the Coin-OR scripts. And I was having trouble with Windows, but it's one of those missing symbol errors which are a pain to track down.